desktop/core/src/desktop/redaction/engine.py

```diff
@@ -106,7 +107,7 @@ def redact(self, message):
     """
 
     if message and (self.trigger is None or self.trigger.search(message)):
-      return self.regex.sub(self.replace, message)
+      return self.regex.sub(smart_str(self.replace), message)
```

Comment:
If you revert the change, does the test fail? :)

Reply:
it has to fail in python-2 as `message` will be processed as `message_to_redact = smart_str(message)` and its a multibyte string.
happen to see the build failing on the first place without abouve change, so i have added those too.
Hopefully no more issue with non-ascii characters from now on in editor.

Reply:
"it has to", so not tested I guess ;)